tutorials/models/04-diffeqbio_II_networkproperties.jmd

```diff
@@ -411,11 +411,11 @@ oprob = ODEProblem(rn, u₀, tspan, p)
 
 We are now ready to solve the problem and plot the solution. Since we have
 essentially generated a method of lines discretization of the diffusion equation
-with a discontinuous initial condition, we'll use an A-L stable implicit ODE
-solver, `KenCarp4`, and plot the solution at a few times:
+with a discontinuous initial condition, we'll use an A stable implicit ODE
```

Comment:
```suggestion
with a discontinuous initial condition, we'll use an A-L stable implicit ODE
```

Reply:
we should probably still mention the L-stability, which is the most important property there.

Reply:
`Rodas5` is L-stable too? (I just ask as the docs, http://docs.juliadiffeq.org/latest/solvers/ode_solve.html, only say that it is A-stable.)

Reply:
yeah it's L-stable, and IIRC it has good B-stability properties as well. We should probably update the docs on that.

Reply:
That's good to know! (Haven't read much about those methods before...) I'll put the L-stability back in and regenerate everything tomorrow when I have a chance.